Explorer/Assets/DCL/InWorldCamera/CameraReelStorageService/Playground/CameraReelRemoteServicesManualTest.cs

```diff
@@ -62,21 +66,56 @@ public async void GetGalleryAsync()
             ThumbnailTexture = await screenshotsStorage.GetScreenshotThumbnailAsync(screenshot.thumbnailUrl);
         }
 
+        [ContextMenu("DELETE IMAGE")]
+        public async void DeleteImageAsync()
+        {
+            Storage = await metadataDatabase.DeleteScreenshotAsync(Result.images.First().id, ct);
+        }
+
         [ContextMenu("UPLOAD IMAGE")]
         public async void UploadImageAsync()
         {
-            CameraReelUploadResponse response = await metadataDatabase.UploadScreenshotAsync(ImageTexture.EncodeToJPG(), Result.images.First().metadata, ct);
+            Metadata.dateTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds().ToString();
+
+            Texture2D readableCopy = MakeReadableCopy(ImageTexture);
+            CameraReelUploadResponse response = await metadataDatabase.UploadScreenshotAsync(readableCopy.EncodeToJPG(), Metadata, ct);
 
             Storage.currentImages = response.currentImages;
             Storage.maxImages = response.maxImages;
 
-            ThumbnailUrl = response.image.thumbnailUrl;
+            ResultThumbnailUrl = response.image.thumbnailUrl;
+
+            Destroy(readableCopy);
```

Comment:
Consider doing a try-finally so the temporary texture is cleaned up even when uploading throws.